File: gr/analog-bubbles.fs
\ analog bubbles (jmcc) #1

8 7.23 2 mce 0 LFSaw.kr 3 * 80 + ( offset )
0.4 0 LFSaw.kr 24 * + MIDICPS ( freq )
0 SinOsc.ar 0.04 * 0.2 0.2 4 CombN
play

( 0.4 0 lfsaw.kr 2 * 8 7.23 mce2 0 lfsaw.kr 0.25 * 9.667 + + exp2
  0 sinosc.ar 0.04 * 0.2 0.2 4 combn play )

File: src/analog-bubbles.hs
-- analog bubbles (jmcc)

import Sound.SC3 {- hsc3 -}

-- > Sound.SC3.UGen.Dot.draw analog_bubbles
analog_bubbles :: UGen
analog_bubbles =
  let o = lfSaw KR (mce2 8 7.23) 0 * 3 + 80
      f = lfSaw KR 0.4 0 * 24 + o
      s = sinOsc AR (midiCPS f) 0 * 0.04
  in combN s 0.2 0.2 4

main :: IO ()
main = audition (out 0 analog_bubbles)
